Makefile

```diff
@@ -26,6 +26,10 @@ endif
 endif
 endif
 
+# bin file definitions
+PHPUNIT=php -d zend.enable_gc=0  vendor/bin/phpunit
```

Comment:
See https://github.com/owncloud/files_antivirus/pull/273 and https://github.com/owncloud/password_policy/pull/166

Now we are running ``phpunit`` from where it is installed in core.